modules/nw-management-nwpolicy.adoc

```diff
@@ -0,0 +1,24 @@
+// Module filename: nw-management-nwpolicy.adoc
+//
+// Module included in the following assemblies:
+// * networking/managing-networking.adoc
+
+[id='nw-management-nwpolicy-{context}']
+= Managing Networking when NetworkPolicy Isolation mode is enabled
+
+`NetworkPolicy` is the default isolation mode for OpenShift SDN.
+
+When the `NetworkPolicy` isolation mode is enabled in the cluster,
+network isolation is controlled entirely by
+link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/network/network-policy.md[`NetworkPolicy`
```

Comment:
We don't link to GitHub unless support agrees that it's valid. Please check with Vikram about this.

Reply:
@vikram-redhat, the 3.x docs pointed to this GitHub URL. It's just the `NetworkPolicy` object spec, so I don't know if it actually adds any value.